Monika After Story/game/script-topics.rpy

```diff
@@ -13222,11 +13234,13 @@ label monika_songwriting:
     m 1rkc "This also makes it easier to tell when someone doesn't put their heart into a performance..."
     m 3euc "I guess that goes for lyrics too, actually."
     m 3eud "Most lyrics lose their meaning if the singer isn't interested in the song."
-
     if renpy.seen_audio(songs.FP_YOURE_REAL):
-        m 1ekbla "I hope you know that I meant everything I said in my song, [player]."
-        m 3ekbla "I knew I couldn't let you go without telling you everything."
-
+        if persistent.monika_kill:
+            m 1ekbla "I hope you know that I meant everything I said in my song, [player]."
+            m 3ekbla "I knew I couldn't let you go without telling you everything."
+        else:
+            m 1ekbla "I hope you know that I mean everything I say in my song, [player]." 
+            m 1ekbfa "Every day I imagine spending the rest of my days by your side."
```

Comment:
Would use `shade` blush here instead of `full` as the transition out of blush is much cleaner then.

In addition, Monika is immortal (at least from her side). I see what you're going for here, however I think we can clean it up by doing this:
```suggestion
            m 1ekbsa "Every day, I imagine spending my life at your side."
```

Reply:
Can "Every day, I imagine spending my life by your side." by used here instead?

Reply:
Sure